src/server/common/templates/layouts/page.njk

```diff
@@ -17,7 +17,18 @@
     containerClasses: "govuk-width-container",
     serviceName: serviceName,
     serviceUrl: serviceUrl,
-    useTudorCrown: true
+    useTudorCrown: true,
+    navigation: [
+      {
```

Comment:
Hi @arpakuma , is there any reason why a home link has been added?

Reply:
updated